src/tests/rest/brp/test_views.py

```diff
@@ -0,0 +1,22 @@
+from unittest import TestCase
+from unittest.mock import patch, MagicMock
+
+from gobstuf.rest.brp.views import (
+    IngeschrevenpersonenView,
+    IngeschrevenpersonenStufResponse,
+    IngeschrevenpersonenStufRequest
+)
+
+
+class TestIngeschrevenpersonenView(TestCase):
+
+    @patch("gobstuf.rest.brp.views.StufRestView", MagicMock())
+    def test_templates_set(self):
+        self.assertEqual(IngeschrevenpersonenStufResponse, IngeschrevenpersonenView.response_template)
+        self.assertEqual(IngeschrevenpersonenStufRequest, IngeschrevenpersonenView.request_template)
+
+    def test_get_not_found_message(self):
+        kwargs = {'bsn': 'BEE ES EN'}
+        self.assertEqual('Ingeschreven persoon niet gevonden met burgerservicenummer BEE ES EN',
+                         IngeschrevenpersonenView().get_not_found_message(**kwargs))
+
```

Comment:
I suggest removing empty test files. It is only confusing.

Reply:
Uhm....

Reply:
Ah I found them. GitHub is placing those comments below files that are not empty